mailpoet/assets/js/src/automation/editor/store/actions.ts

```diff
@@ -11,6 +11,24 @@ import { LISTING_NOTICE_PARAMETERS } from '../../listing/workflow-listing-notice
 import { MailPoet } from '../../../mailpoet';
 import { WorkflowStatus } from '../../listing/workflow';
 
+const trackErrors = (errors) => {
+  if (!errors?.steps) {
+    return;
+  }
+  const payload = Object.keys(errors.steps as object).map((stepId) => {
+    const error = errors.steps[stepId];
+    const stepKey = select(storeName).getStepById(stepId)?.key;
+    const fields = Object.keys(error.fields as object)
+      .map((field) => `${stepKey}/${field}`)
+      .reduce((prev, next) => prev.concat(next));
```

Comment:
Is this line equal to `.flatten()`, maybe? Not sure if my brain processes it correctly now.

Reply:
![image](https://user-images.githubusercontent.com/6458412/200491839-b6ab7751-e02f-43fc-a4e6-ad99095e13c7.png)
yes. it is equal.